environment.yml

```diff
@@ -27,7 +27,7 @@ dependencies:
   - click
   - autograd
   - absl-py
-  - dask
+  - dask[complete]
```

Comment:
Why is `dask[complete]` needed for the conda environment? Just using `dask` in the environment file works fine for me. I think `dask[complete]` is just needed for the pip install.